File: Zestawy Zadań/Python/5 - Funkcje, rekurencja/Zestaw 1 zadania/5.4 Funkcja o prototypie int MODROZ(int a, int b).py
def MODROZ(a, b):
    return abs(a - b)

if __name__ == "__main__":
    X = int(input("Podaj pierwszą liczbę całkowitą: "))
    Y = int(input("Podaj drugą liczbę całkowitą: "))
    print("Wynik:", MODROZ(X, Y))

File: Zestawy Zadań/C++/5 - Funkcje, rekurencja/Zestaw 2 zadania/5.4  Funkcja o prototypie int MODROZ(int a, int b).cpp
#include <iostream>
#include <cstdlib>

int MODROZ(int a, int b) {
    return std::abs(a - b);
}

int main() {
    int X, Y;
    std::cout << "Podaj pierwsz¹ liczbê ca³kowit¹: ";
    std::cin >> X;
    std::cout << "Podaj drug¹ liczbê ca³kowit¹: ";
    std::cin >> Y;
    std::cout << "Wynik: " << MODROZ(X, Y) << std::endl;
    return 0;
}
